src/components/Input/Input.jsx

```diff
@@ -0,0 +1,25 @@
+import cn from "classnames";
+import "./Input.scss";
+
+export default function Input({
+  name,
+  value,
+  placeholder,
+  textPosition,

```

Comment:
Sweet! 

Just the name, you can mimic the property used in CSS rules:

```suggestion
  textAlign,
```